Assets/Code/Interactables/Cards/Card.cs

```diff
@@ -11,15 +11,15 @@ public class Card {
     public int lifeCost { get; private set; }
     public bool hasTargets { get; private set; }
     // this is the list of game effects to perform when a card is played
-    public List<PlayEffect> Effects { get; private set; }
+    public List<DynamicEffect> Effects { get; private set; }
 
     // Visual component of the card, stored within its own View class
     private CardView display;
     private GameObject prefab;
 
     // Constructor that creates the object, but does not instantiate visuals.
     // Those can be called as needed by the CreateVisual() function
-    public Card(string name, int id, string description, int cost, bool hasTargets, List<PlayEffect> effects) {
+    public Card(string name, int id, string description, int cost, bool hasTargets, List<DynamicEffect> effects) {
```

Comment:
missed this before but i think we don't need hasTargets anymor now that we have TargetableDynamicEffects

Reply:
Agreed, I forgot about it as well. Will remove.